wp-content/themes/orient-theme/js/script.js

```diff
@@ -124,6 +124,27 @@ if(jQuery('.content aside').length) {
 	});
 }
 
+var disableCommentSubmission = function() {
```

Comment:
This function should maybe not be named `disableCommentSubmission` since that's not the only thing it's doing. It's not just disabling, it's checking and displaying word count and disabling if the comment doesn't fit the policy.